internal/cmd/dev.go

```diff
@@ -1,206 +1,85 @@
 package cmd
 
 import (
-	"bytes"
-	"encoding/json"
 	"fmt"
-	"github.com/gin-gonic/gin"
-	"net/http"
+	"os"
+	"os/exec"
+	"os/signal"
+	"path/filepath"
+	"syscall"
 
-	"database/sql"
 	"github.com/chiselstrike/iku-turso-cli/internal"
 	"github.com/spf13/cobra"
-	"io"
-	_ "modernc.org/sqlite"
```

Comment:
We probably need `go mod tidy` to remove unused dependencies.

Reply:
should we add to CI a test that requires go mod tidy to come empty?